met/src/tools/other/pb2nc/pb2nc.cc

```diff
@@ -3211,6 +3211,12 @@ void interpolate_pqtzuv(float *prev_pqtzuv, float *cur_pqtzuv, float *next_pqtzu
 
 ////////////////////////////////////////////////////////////////////////
 
+static bool is_valid_pb_data(float pb_value) {
+   return (!is_eq(pb_value,bad_data_float) && pb_value < r8bfms);
```

Comment:
Recommend calling "is_bad_data(float)" function instead which also checks for NaN.
```suggestion
   return (!is_bad_data(pb_value) && pb_value < r8bfms);
```